kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -159,11 +159,19 @@ public void run() {
             boolean handleBrokerState() {
                 LOGGER.trace("Polling {}", brokerStateName);
                 boolean ready = false;
-                Integer running = Integer.valueOf(3);
-                Object value = brokerState.value();
+                Integer runningState = 3;
+                Integer unknownState = 127;
```

Comment:
Would it make sense to have these as some static final constants?

Reply:
Yes, there's no reason for them to be set here